tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,293 @@
-##############################################################################
-# Now, let's compute dSPM on a raw file within a label:
-
-fname_label = data_path + '/MEG/sample/labels/Aud-lh.label'
-label = mne.read_label(fname_label)
-
-##############################################################################
-# Compute inverse solution during the first 15s:
-
-from mne.minimum_norm import apply_inverse_raw  # noqa
-start, stop = raw.time_as_index([0, 15])  # read the first 15s of data
-stc = apply_inverse_raw(raw, inverse_operator, lambda2, method, label,
-                        start, stop)
-
-##############################################################################
-# Save result in stc files:
-
-stc.save('mne_dSPM_raw_inverse_Aud')
-
-##############################################################################
-# What else can you do?
-# ^^^^^^^^^^^^^^^^^^^^^
-#
-#     - detect heart beat QRS component
-#     - detect eye blinks and EOG artifacts
-#     - compute SSP projections to remove ECG or EOG artifacts
-#     - compute Independent Component Analysis (ICA) to remove artifacts or
-#       select latent sources
-#     - estimate noise covariance matrix from Raw and Epochs
-#     - visualize cross-trial response dynamics using epochs images
-#     - compute forward solutions
-#     - estimate power in the source space
-#     - estimate connectivity in sensor and source space
-#     - morph stc from one brain to another for group studies
-#     - compute mass univariate statistics base on custom contrasts
-#     - visualize source estimates
-#     - export raw, epochs, and evoked data to other python data analysis
-#       libraries e.g. pandas
-#     - and many more things ...
-#
-# Want to know more ?
-# ^^^^^^^^^^^^^^^^^^^
-#
-# Browse :ref:`the examples gallery <sphx_glr_auto_examples>`.
-
-print("Done!")
+# Finally, we can estimate the cortical origins of the evoked activity by
+# projecting the sensor data into this subject's :term:`source space`.
+# MNE-Python supports lots of ways of doing this (minimum-norm estimation,
+# dipole fitting, beamformers, etc); here we'll use minimum-norm estimation
+# (MNE) to generate a continuous cortical activation map. To do this we'll
+# need the inverse operator for this subject (the sample data includes one
+# that's been pre-computed), and we'll need to specify a regularization
+# parameter:
+
+inverse_operator_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                     'sample_audvis-meg-oct-6-meg-inv.fif')
+inv_operator = mne.minimum_norm.read_inverse_operator(inverse_operator_file)
+# set the regularization parameter (λ²)
+signal_to_noise_ratio = 3.
+regularization_param = 1. / signal_to_noise_ratio ** 2
+# generate the STC
+source_time_course = mne.minimum_norm.apply_inverse(vis_evoked, inv_operator,
+                                                    regularization_param,
+                                                    method='MNE')
```

Comment:
I would try to advertise a bit the software architecture. For e.g., here you can mention all the different options that there for method and maybe even show that by changing the method from 'MNE' to 'dspm' one can very easily try different source localization methods. Similarly, all plotting functions start with plot_ and that they all return a figure handle which can be easily manipulated in matplotlib. The point is to make people _believe_ that they can do a whole lot by learning just a little.

Reply:
I'll add the mention of other source localization methods here.  The "returns figure handle" is already shown above when we call `plot_events`.